remoteappmanager/static/js/vue/filters.js

```diff
@@ -1,20 +1,20 @@
 define([
     "components/vue/dist/vue",
     "urlutils"
-], function (Vue, urlutils) {
+], function (Vue, urlUtils) {
```

Comment:
I just wanted to stay consistent in all files (except for the variables which come from the server). But if I rename it `urlUtils` I should maybe rename the file as well ?